modules/storage-persistent-storage-pv.adoc

```diff
@@ -152,6 +153,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |iSCSI  | ✅ | ✅ |  -
 |Local volume | ✅ | - |  -
 |NFS  | ✅ | ✅ | ✅
+|OpenStack Manila  | ✅ | ✅ | ✅
```

Comment:
Basing this on "Access rules" section here: https://docs.openstack.org/manila/latest/contributor/driver_requirements.html

Reply:
@bobfuru From the test, manila csi driver can not support `RWO` and `ROX`, there is a bug to track it. https://bugzilla.redhat.com/show_bug.cgi?id=1871708

Reply:
Removed RWO and ROX Manila support from the table.